modules/planning/lattice/trajectory_generation/trajectory1d_generator.cc

```diff
@@ -137,16 +141,10 @@ void Trajectory1dGenerator::GenerateLongitudinalTrajectoryBundle(
 void Trajectory1dGenerator::GenerateLateralTrajectoryBundle(
     std::vector<std::shared_ptr<Curve1d>>* ptr_lat_trajectory_bundle) const {
   auto end_conditions = end_condition_sampler_.SampleLatEndConditions();
-  for (const auto& end_condition : end_conditions) {
-    auto ptr_trajectory1d = std::make_shared<LatticeTrajectory1d>(
-        std::shared_ptr<Curve1d>(new QuinticPolynomialCurve1d(
-            init_lat_state_, end_condition.first, end_condition.second)));
 
-    ptr_trajectory1d->set_target_position(end_condition.first[0]);
-    ptr_trajectory1d->set_target_velocity(end_condition.first[1]);
-    ptr_trajectory1d->set_target_time(end_condition.second);
-    ptr_lat_trajectory_bundle->push_back(ptr_trajectory1d);
-  }
+  // Use the common function to generate trajectory bundles.
+  GenerateTrajectoryBundlesWithCondition(init_lat_state_, end_conditions,
```

Comment:
Should not use the common function here, because quintic polynomial is used here

Reply:
The common function uses different initial and end conditions to generate curve bundles. So it can be used here. Please check the following line in the common function  carefully.  
```
auto ptr_trajectory1d = std::make_shared<LatticeTrajectory1d>(
        std::shared_ptr<Curve1d>(new QuarticPolynomialCurve1d(
            init_condition, end_state, end_condition.second)));
```